### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by SmritiSatya**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by wei-harness — COMMENTED** [review]
@SmritiSatya could you please explain more about this request?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -7,8 +7,8 @@
 *       @pratmit @wei-harness
 
 # Configuration Files in DS are JS.
-*.js    @wei-harness @rohanmaharjan100
-*.ts    @wei-harness @rohanmaharjan100
+*.js    @wei-harness @rohanmaharjan100 @ajeshbaby
```
Comment (pratmit) [inline .github/CODEOWNERS:10]: What is the reason for this change?

Comment (SmritiSatya) [reply .github/CODEOWNERS:10]: https://harness.slack.com/archives/C8LE2H4BE/p1729008426548489?thread_ts=1728985147.631389&cid=C8LE2H4BE -  reference conversation
I am working on restructuring the docs for one section, and it has some link changes to .js and .ts files. So, I thought it would be better to add my manager to the codeowner for this section instead of myself. 
What do you think @ravilach ?

Comment (SmritiSatya) [reply .github/CODEOWNERS:10]: Actually, don't mind. Closing this. @wei-harness @pratmit
